readthedocs/api/base.py

```diff
@@ -223,7 +221,11 @@ def get_anchor(self, request, **kwargs):
         self.throttle_check(request)
 
         query = request.GET.get('q', '')
-        redis_data = redis_client.keys("*redirects:v4*%s*" % query)
+        try:
+            redis_client = cache.get_client(None)
```

Comment:
It's exposed from `redis_cache`. If the cache backend isn't redis, this expects to catch the `AttributeError` that will be thrown